addons/mail/static/src/core/web/chatter.js

```diff
@@ -199,13 +202,16 @@ export class Chatter extends Component {
      */
     get toFollowersText() {
         const threadName = this.state.thread.displayName || this.state.thread.name;
+        const followers = this.state.thread.followers.slice(-5).map(({ partner }) =>
+            sprintf('<span title="%(email)s">%(emailWithoutDomain)s</span>', {
+                email: escape(partner.email),
+                emailWithoutDomain: escape(partner.emailWithoutDomain),
+            })
+        );
         const toFollowersText = threadName
-            ? sprintf(
-                  _t(
-                      '<span class="fw-bold">To:</span> <span class="fst-italic">Followers of</span> <span class="fw-bold">"%(thread name)s"</span>.'
-                  ),
-                  { "thread name": escape(threadName) }
-              )
+            ? sprintf(_t('<span class="fw-bold">To:</span> %(followers)s.'), {
+                  followers: followers.join(", "),
+              })
```

Comment:
Note that `, ` is local-dependent, e.g. it's `、` in Japanese.
You might use `Intl.ListFormat(str, { type: "conjunction" })` to properly handle l18n